src/graphql/entityDescriptor.ts

```diff
@@ -194,6 +194,9 @@ export class FieldConfig {
 
   @Field((type) => UpdateRules, { nullable: true })
   public updateRules?: UpdateRules;
+
+  @Field((type) => String, { nullable: true })
+  public lookupTable: string;
```

Comment:
add ts typing for nullable field:

 @Field((type) => String, { nullable: true })
  public lookupTable?: string;